src/routers/__tests__/TabRouter-test.js

```diff
@@ -600,4 +602,60 @@ describe('TabRouter', () => {
       ],
     });
   });
+
+  test('Handles the tab selection action when moving to tabrouter', () => {
+    const ChildNavigator = () => <div />;
+    ChildNavigator.router = TabRouter({
+      FooTabs: BareLeafRouteConfig,
```

Comment:
please use singular: `FooTab`

Reply:
yup, just one tab. fixed in the latest commit.